src/library.js

```diff
@@ -3548,6 +3548,9 @@ mergeInto(LibraryManager.library, {
 #if hasExportedSymbol('emscripten_builtin_memalign')
     size = alignMemory(size, {{{ WASM_PAGE_SIZE }}});
     var ptr = _emscripten_builtin_memalign({{{ WASM_PAGE_SIZE }}}, size);
+#if CAN_ADDRESS_2GB
+    ptr >>>= 0;
+#endif
```

Comment:
Why does this location need it, but not `FS.read` right below in the diff?

(I guess this is a JS function, so no signature info to let us automatically do this - but the same is true of `FS.read`, isn't it?)

Reply:
Because this is a pointer being returns from a native/wasm function.. not a pointer arriving as an argument to a JS library function.  This PR automates convertion of the latter.. I have a followup to automate the former.

Reply:
How do you know that `ptr` is returned from a wasm function? Are you looking at all callers to this function?

Reply:
The variable is declared on the line above as the result of a call to native function.